### ISSUE TITLE
The styling for the "Become a Prime Sponsor" button is not correct.

### ISSUE THREAD

**Issue by the-shivam-gupta** [reactplay/react-play#1268]
When the button is hovered over, the svg contained within it overlaps the text.

![image](https://github.com/reactplay/react-play/assets/109647722/dd1d904b-3220-4e46-a83d-5666a7d4dc3f)

**Comment by priyankarpal** [comment]
which resolution? @the-shivam-gupta

**Comment by the-shivam-gupta** [comment]
Starting from 800 pixels onwards

**Comment by priyankarpal** [comment]
got it.

**Comment by SAGARSINGH-1** [comment]
hey @priyankarpal please assign this issue to me

**Comment by priyankarpal** [comment]
> hey @priyankarpal please assign this issue to me

I've assigned to @the-shivam-gupta

**Comment by ayushsgithub** [comment]
Hey @the-shivam-gupta, are you still working on this issue?

**Comment by the-shivam-gupta** [comment]
Yes I'm I have done it earlier but I was getting errors while downloading the npm package.

**Comment by priyankarpal** [comment]
> Yes I'm I have done it earlier but I was getting errors while downloading the npm package.

please share the issue

**Comment by the-shivam-gupta** [comment]
After editing the file, when I run the project, the environment shows a "kill -9" message.

**Comment by priyankarpal** [comment]
> After editing the file, when I run the project, the environment shows a "kill -9" message.

solved?

**Comment by the-shivam-gupta** [comment]
No, it's still happening. You can assign it to someone else. I'm facing many issues while running.

**Comment by t-trafer** [comment]
@priyankarpal i would like to fix this issue if this is available

**Comment by priyankarpal** [comment]
> @priyankarpal i would like to fix this issue if this is available

someone already created a PR :)

**Comment by letscodedanish** [comment]
Fixed the issue with some small changes in code. Efficiently.
